src/components/va-alert/va-alert.tsx

```diff
@@ -106,21 +111,21 @@ export class VaAlert {
   }
 
   render() {
-    const { backgroundOnly, status, visible, closeable } = this;
+    const { backgroundOnly, status, visible, closeable , hideContent} = this;
     const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
 
-    if (!visible) return <div aria-live="polite" />;
-
+    if (!visible) return <div aria-live="polite" />; 
+    
     return (
       <Host>
         <div class={classes}>
           <i aria-hidden="true" role="img"></i>
           <span class="sr-only">Alert: </span>
           <div class="body" onClick={this.handleAlertBodyClick.bind(this)}>
             {!backgroundOnly && <slot name="headline"></slot>}
-            <div class="text">
+            {!hideContent && <div class="text">
```

Comment:
I think we can simplify this by not adding the extra `hideContent` prop. Since the `text` class only served as a target for the CSS, we can remove that div entirely and change how we apply the margin, like so:

![image](https://user-images.githubusercontent.com/2008881/122267023-49e01700-ce8f-11eb-81c4-7d32165178e8.png)


Now, if I put this on a page:

```html
    <va-alert status="success">
      <h3 slot="headline">Congrats, you did it!</h3>
    </va-alert>
    <br />
    <va-alert status="success" background-only>
      <div>Some content</div>
    </va-alert>
 ```
 
 I get what we expect:
 
 
![image](https://user-images.githubusercontent.com/2008881/122267319-97f51a80-ce8f-11eb-889b-13282d849341.png)

This implementation works in IE11 as well thanks to the CSS polyfills:

![signal-2021-06-16-104548](https://user-images.githubusercontent.com/2008881/122267907-48fbb500-ce90-11eb-82f2-d63b11041119.png)

Reply:
That works for me @bkjohnson! I will update this PR to be just those changes :-)